src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/create-rgw-service-entities/create-rgw-service-entities.component.html

```diff
@@ -1,70 +1,138 @@
-               for="zonegroupName"
-               i18n>Zone Group Name</label>
-        <div class="cd-col-form-input">
-          <input class="form-control"
-                 type="text"
-                 placeholder="Zone group name..."
-                 id="zonegroupName"
-                 name="zonegroupName"
-                 formControlName="zonegroupName">
-          <span class="invalid-feedback"
-                *ngIf="createMultisiteEntitiesForm.showError('zonegroupName', formDir, 'required')"
-                i18n>This field is required.</span>
-        </div>
+
+      <div class="form-item">
+        <cds-text-label
+          labelInputID="zonegroupName"
+          [invalid]="
+            createMultisiteEntitiesForm.controls.zonegroupName.invalid &&
+            (createMultisiteEntitiesForm.controls.zonegroupName.touched ||
+              createMultisiteEntitiesForm.controls.zonegroupName.dirty)
+          "
+          [invalidText]="zonegroupNameError"
+          cdRequiredField="Zone Group Name"
+          i18n
+          >Zone Group Name
+          <input
+            cdsText
+            type="text"
+            placeholder="Zone group name..."
+            id="zonegroupName"
+            name="zonegroupName"
+            formControlName="zonegroupName"
+            [invalid]="
+              createMultisiteEntitiesForm.controls.zonegroupName.invalid &&
+              (createMultisiteEntitiesForm.controls.zonegroupName.touched ||
+                createMultisiteEntitiesForm.controls.zonegroupName.dirty)
+            "
+          />
+        </cds-text-label>
+        <ng-template #zonegroupNameError>
+          <span
+            class="invalid-feedback"
+            *ngIf="createMultisiteEntitiesForm.showError('zonegroupName', formDir, 'required')"
+            i18n
+            >This field is required.</span
+          >
+        </ng-template>
       </div>
-      <div class="form-group row">
-        <label class="cd-col-form-label required"
-               for="zoneName"
-               i18n>Zone Name</label>
-        <div class="cd-col-form-input">
-          <input class="form-control"
-                 type="text"
-                 placeholder="Zone name..."
-                 id="zoneName"
-                 name="zoneName"
-                 formControlName="zoneName">
-          <span class="invalid-feedback"
-                *ngIf="createMultisiteEntitiesForm.showError('zoneName', formDir, 'required')"
-                i18n>This field is required.</span>
-        </div>
+
+      <div class="form-item">
+        <cds-text-label
+          labelInputID="zoneName"
+          [invalid]="
+            createMultisiteEntitiesForm.controls.zoneName.invalid &&
+            (createMultisiteEntitiesForm.controls.zoneName.touched ||
+              createMultisiteEntitiesForm.controls.zoneName.dirty)
+          "
+          [invalidText]="zoneNameError"
+          cdRequiredField="Zone Name"
```

Comment:
Since all field are required we can add the default helper text below heading , see `cd-help-text`